docs/kotlin/getting-started.md

```diff
@@ -107,11 +107,11 @@ We will use [_remote plugins_][remote-plugins], a feature of the [Buf Schema Reg
 how to generate code by putting this into [`buf.gen.yaml`][buf.gen.yaml]:
 
 ```yaml title=buf.gen.yaml
-version: v1
+version: v2
 plugins:
-  - plugin: buf.build/protocolbuffers/java
+  - remote: buf.build/protocolbuffers/java
     out: app/src/main/java
-  - plugin: buf.build/connectrpc/kotlin
+  - remote: buf.build/connectrpc/kotlin
```

Comment:
Our recommendation is to put both `buf.yaml` and `buf.gen.yaml` at the root of the repository going forward, so I think these instructions need to be rewritten as:

```
$ buf config init
```

Then edit `buf.yaml` to add:
```yaml
modules:
  - path: proto
```

Then run:
```
$ buf export buf.build/connectrpc/eliza -o proto
```

Then we'd remove the `cd ..` step and the `buf generate proto` command can become just `buf generate`.